Jenkinsfile_CNP

```diff
@@ -24,12 +29,44 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
   ]
 }
 
+env.RESOURCE_GROUP = 'bulk-scan-aks-rg'
```

Comment:
don't add the `-rg` suffix its of type resource group, no need to put the type in the name

Reply:
Removed